### ISSUE TITLE
train.py file doesn’t exist

### ISSUE THREAD

**Issue by ssatz** [whn09/table_structure_recognition#5]
Hi, thanks for the repo. I am trying to train the pub1m table structure on yolo8. I couldn’t find the train.py file. 
Label table row accuracy is no at all good. Any suggestions

**Comment by whn09** [comment]
I used Yolov5 to train the model, but I think Yolov8 is similar

**Comment by ssatz** [comment]
hi thanks this has been solved. I looked into wrong Yolo version. 
I think you didn't train the Fintab Dataset?

**Comment by whn09** [comment]
Yes, I didn't train the Fintab Dataset. But maybe you can merge the dataset with PubTables-1M. I also recommend you to set Yolo `size` to at least 800 to get best performance.

**Comment by ssatz** [comment]
Yes I just trained for fintab with 640 size and table detection is not good and getting short rows. I will try it with 800.

**Comment by ssatz** [comment]
I trained  both  Fintab and PubTables- 1m and normal structure the results are good and i think need to train PaddleOCR for text detection. 

Here is the Yolo format data **https://huggingface.co/datasets/Codeplug/pub-fintab-yolov**
**Metrics:**

![image](https://github.com/ultralytics/ultralytics/assets/6892242/2e0b9a66-4d57-41ed-b9d2-ca38556e709c)

<img width="427" alt="image" src="https://github.com/ultralytics/ultralytics/assets/6892242/c1574bc6-68be-4850-b547-edc0d4aeed9c">

For training the dataset we have used [Runpod](https://runpod.io?ref=b07ucj1b)
